django/contrib/postgres/constraints.py

```diff
@@ -155,12 +155,13 @@ def __eq__(self, other):
         return super().__eq__(other)
 
     def __repr__(self):
-        return '<%s: index_type=%s, expressions=%s%s%s%s%s>' % (
+        return '<%s: index_type=%s name=%s expressions=%s%s%s%s%s>' % (
             self.__class__.__qualname__,
-            self.index_type,
-            self.expressions,
-            '' if self.condition is None else ', condition=%s' % self.condition,
-            '' if self.deferrable is None else ', deferrable=%s' % self.deferrable,
-            '' if not self.include else ', include=%s' % repr(self.include),
-            '' if not self.opclasses else ', opclasses=%s' % repr(self.opclasses),
+            repr(self.index_type),
+            repr(self.name),
+            repr(self.expressions),
+            '' if self.condition is None else ' condition=%s' % str(self.condition),
+            '' if self.deferrable is None else ' deferrable=%s' % str(self.deferrable),
```

Comment:
Type of `condition` and `deferrable` is checked in `__init__()` so `str()` is not necessary.